src/content/docs/docs/tool-calling.mdx

```diff
@@ -205,6 +205,35 @@ Include defined tools in your prompts to generate content.
   </TabItem>
 </Tabs>
 
+### Streaming and Tool Calling
+
+When combining tool calling with streaming responses, you will receive `toolRequest` and `toolResponse` content parts in the chunks of the stream. For example, the following code:
+
+```ts
+const {stream, output} = ai.generateStream({
```

Comment:
It's not `output` but `response`, but since you're now using it as all might as well remove it.

```suggestion
const { stream } = ai.generateStream({
```